web/portal/brand/src/entities/DestinationRateGroup/Action/ImportRates.tsx

```diff
@@ -0,0 +1,225 @@
+import { EmbeddableProperty, EntityValidatorResponse } from '@irontec/ivoz-ui';
+import { MoreMenuItem } from '@irontec/ivoz-ui/components/List/Content/Shared/MoreChildEntityLinks';
+import { StyledTableRowCustomCta } from '@irontec/ivoz-ui/components/List/Content/Table/ContentTable.styles';
+import {
+  OutlinedButton,
+  SolidButton,
+} from '@irontec/ivoz-ui/components/shared/Button/Button.styles';
+import { useFormHandler } from '@irontec/ivoz-ui/entities/DefaultEntityBehavior/Form/useFormHandler';
+import {
+  ActionFunctionComponent,
+  ActionItemProps,
+} from '@irontec/ivoz-ui/router/routeMapParser';
+import { FileUploadFactory } from '@irontec/ivoz-ui/services/form/FormFieldFactory/Factory/FileUploadFactory';
+import _ from '@irontec/ivoz-ui/services/translations/translate';
+import UploadFileIcon from '@mui/icons-material/UploadFile';
+import {
+  Box,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  styled,
+  Tooltip,
+} from '@mui/material';
+import Papa from 'papaparse';
+import { useEffect, useState } from 'react';
+import { useStoreActions, useStoreState } from 'store';
+
+import DestinationRateGroup from '../DestinationRateGroup';
+import ImportRatesMappingTable from './ImportRatesMappingTable';
+
+export const StyledDialog = styled(Dialog)(() => {
+  return {
+    '& .MuiPaper-root': {
+      maxWidth: '90%',
+    },
+    '&.width .MuiPaper-root': {
+      width: '90%',
+    },
+  };
+});
+
+const ImportRates: ActionFunctionComponent = (props: ActionItemProps) => {
+  const { row, entityService, variant = 'icon' } = props;
+  const formik = useFormHandler({
+    create: false,
+    entityService,
+    fixedValues: {},
+    filterValues: {},
+    filterBy: undefined,
+    initialValues: row,
+    validator: (values) => values as EntityValidatorResponse,
+    onSubmit: async () => {
+      /* noop */
```

Comment:
is this method needed here? It can't be optional?

Reply:
It's just to avoid "empty function" linter error. I prefer this rather than eslint-disable comment